admin/spec/back-end/service/attendance-service.spec.js

```diff
@@ -162,4 +162,40 @@ describe('attendanceService', () => {
       done()
     })
   })
+
+  describe('#hasAttendance', () => {
+    describe('for live env', () => {
+      it('returns valid if pupil has any attendance', async () => {
+        spyOn(pupilAttendanceDataService, 'findOneByPupilId').and.returnValue({ id: 'id', attendanceCode_id: 100 })
+        const result = await service.hasAttendance('id', 'live')
+        expect(result).toBeTruthy()
```

Comment:
I would prefer this test to actually test for the boolean that is returned from `hasAttendance()`.

Reply:
Good catch - updated.